disnake/http.py

```diff
@@ -1038,7 +1038,7 @@ def start_thread_without_message(
         channel_id: Snowflake,
         *,
         name: str,
-        auto_archive_duration: threads.ThreadArchiveDuration,
+        auto_archive_duration: threads.ThreadArchiveDurationT,
```

Comment:
```suggestion
        auto_archive_duration: threads.ThreadArchiveDuration,
```
also shouldn't change